src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private final double upperSide;
+    private final double bottomSide;
+    private final double height;
+
+    public IsoscelesTrapezoid(String color, double upperSide, double bottomSide, double height) {
+        super(color);
+        this.upperSide = upperSide;
+        this.bottomSide = bottomSide;
+        this.height = height;
+    }
+
+    @Override
+    public double getArea() {
+        return ((bottomSide + upperSide) / 2) * height;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: isosceles trapezoid," + " area: "
+                + getArea() + " sq. units, bottom side: "
+                + bottomSide + " units, upper side: "
+                + upperSide + " units, height: "
+                + height + " units, color: " + super.getColor());
```

Comment:
Avoid using magic numbers in your code. In this case, 'isosceles trapezoid' is a string literal that appears to be used as a figure type identifier. It's better to define it as a constant, especially if it's used in multiple places or might change in the future.